allocated-pids.txt

```diff
@@ -682,4 +682,5 @@ PID    | Product name
 0x82A2 | RobotClass Graphite S3 - CircuitPython
 0x82A3 | RobotClass Graphite S3 - UF2 Bootloader
 0x82A4 | ENERTY Module M - UF2 Bootloader
-0x82A5 | OpenRemise - S3Main
\ No newline at end of file
+0x82A5 | OpenRemise - S3Main
+0x82A6 | Espressif zigbee dongle
```

Comment:
```suggestion
0x82A6 | Espressif Zigbee NCP Dongle
```

Reply:
Close it because of it's used for customer.